configs/mistral-7b/finetuning/olive-config.json

```diff
@@ -2,16 +2,46 @@
     "input_model":{
         "type": "PyTorchModel",
         "config": {
+            "model_script": "finetuning/qlora_user_script.py",
+            "io_config": "get_merged_decoder_with_past_io_config",
+            "dummy_inputs_func": "get_merged_decoder_with_past_dummy_inputs",
```

Comment:
These 3 lines are the same across all files. The model_script property seems necessary, but are the other 2 (io_config and dummy_inputs_func) needed?